system/sleep/README.md

```diff
@@ -0,0 +1,56 @@
+# Logger
+It is a helper library that allows logging to different streams using different loggers simultaneously. 
+This removes the burden of choosing the appropriate logging service from the wrapper developer, and shifts it to the app developer that wants to integrate the wrapper.
+
```

Comment:
Please update README :)

Reply:
Good catch. Thank you! I updated this one and also made some minor updates to the wrapper and plugin readmes.